src/app/manager-dashboard/reports/reports-myplanet.component.scss

```diff
@@ -1,7 +1,25 @@
 @import '../../variables';
 
+.responsive-toolbar div {
+    display: flex;
+    align-items: center;
+}
+
 @media (max-width: $screen-sm) {
-    .search-bar {
-        overflow: hidden;
-    }    
+    .responsive-toolbar {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        width: 100%;
+        height: auto;
+    }
+    
+    .left-offset {
```

Comment:
You can combine these css properties together since they both fall under .left-offset {}

Reply:
oh yea oops, thanks for catching that